styles.css

```diff
@@ -0,0 +1,164 @@
+@media only screen and (max-width: 600px) {
```

Comment:
Please use mobile-first design. You can do this by designing the page for mobile devices first and then for desktops using min-width for the media queries. (Please move this to the bottom and change max-width to min-width.) This is to reduce load time on these devices since users use them to connect to websites more than desktops.